assignment-client/src/messages/MessagesMixer.cpp

```diff
@@ -62,13 +69,19 @@ void MessagesMixer::handleMessages(QSharedPointer<ReceivedMessage> receivedMessa
 void MessagesMixer::handleMessagesSubscribe(QSharedPointer<ReceivedMessage> message, SharedNodePointer senderNode) {
     QString channel = QString::fromUtf8(message->getMessage());
     _channelSubscribers[channel] << senderNode->getUUID();
+
+    _allSubscribers[senderNode->getUUID()] << 0;
```

Comment:
```suggestion
    _allSubscribers[senderNode->getUUID()] = 0;
```

Reply:
I don’t think you need this line.  the timer clears it every second, and the message received code above handles when it’s not in there